pom.xml

```diff
@@ -27,7 +27,7 @@
         <version>29</version>
     </parent>
 
-    <name>Keycloak</name>
+    <name>Keycloak - Open Hous</name>
```

Comment:
This looks like a typo "Hous" . Could you please fix it?

Reply:
Fixed